components/spluseins-footer.vue

```diff
@@ -13,35 +13,47 @@
         text-xs-center
         white--text
         xs12>
-        &copy;2018 — <strong>sPlusEins</strong>
+        &copy;2018 — 
+        <strong 
+          class="about"
+          @click="activeAbout = true">
+          SplusEins
+        </strong>
       </v-flex>
     </v-layout>
+    <spluseins-about
+      :active = "activeAbout" 
+      class="about" 
+      @close="activeAbout = false"/>
   </v-footer>
 
 </template>
 
 <script lang="js">
-  export default  {
-    name: 'SplusEinsFooter',
-    data() {
-      return {
-
-      }
-    },
-    computed: {
+import SpluseinsAbout from './spluseins-about.vue'
+
+export default  {
+  name: 'SplusEinsFooter',
+  components: {SpluseinsAbout},
+  data() {
+    return {
+      activeAbout : false
```

Comment:
`aboutActive` wäre besser, weil man booleans in der Regel als "is about active" bezeichnet.
(Bsp. https://www.htmlgoodies.com/html5/javascript/back-by-popular-demand-more-on-javascript-variable-naming-conventions.html)